### PR TITLE
Update TH permissions

### PR DESCRIPTION
**Pull request by annaliseirby090**
Update TH permissions to add `isCaseContactOwner`

### CODE REVIEW

File: hrm-domain/hrm-core/permission-rules/th.json
Comment (janorivera) [inline hrm-domain/hrm-core/permission-rules/th.json:19]: I think editChildIsAtRisk and editFollowUpDate are no longer needed. 
@stephenhand  can confirm.
Besides that it lgtm, but I've been told that this repo needs explicit approval from an engineer :)

Comment (stephenhand) [reply hrm-domain/hrm-core/permission-rules/th.json:19]: Yes those 2 are redundant and should be removed. We should probably just do a sweep and remove them from all accounts in both projects, to prevent them keep popping up again

**Review by stephenhand — APPROVED** [review]
Approving assuming the 2 redundant actions will be removed

**Pull request merged**
